client-flink/flink1.14/src/main/java/org/apache/celeborn/plugin/flink/FlinkResultPartitionInfo.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.plugin.flink;
```

Comment:
spark plugin's package name is "org.apache.spark.shuffle.celeborn"
do package name keep the same pattern with spark plugin?
flink plugin package name cane be "org.apache.flink.shuffle.celeborn"

Reply:
It's no need to keep the name pattern like spark plugin as spark only allow access shuffle class from specified package but Flink does not have this limitation as I know.